docs/_source/tutorials/notebooks/labelling_textclassification_setfit_zeroshot.ipynb

```diff
@@ -0,0 +1,645 @@
+              "                            \n",
+              "var gd = document.getElementById('40e1c2c5-21e1-4fd4-904b-042e99d5fd36');\n",
+              "var x = new MutationObserver(function (mutations, observer) {{\n",
+              "        var display = window.getComputedStyle(gd).display;\n",
+              "        if (!display || display === 'none') {{\n",
+              "            console.log([gd, 'removed!']);\n",
+              "            Plotly.purge(gd);\n",
+              "            observer.disconnect();\n",
+              "        }}\n",
+              "}});\n",
+              "\n",
+              "// Listen for the removal of the full notebook cells\n",
+              "var notebookContainer = gd.closest('#notebook-container');\n",
+              "if (notebookContainer) {{\n",
+              "    x.observe(notebookContainer, {childList: true});\n",
+              "}}\n",
+              "\n",
+              "// Listen for the clearing of the current output cell\n",
+              "var outputEl = gd.closest('.output');\n",
+              "if (outputEl) {{\n",
+              "    x.observe(outputEl, {childList: true});\n",
+              "}}\n",
+              "\n",
+              "                        })                };                            </script>        </div>\n",
+              "</body>\n",
+              "</html>"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        }
+      ],
+      "source": [
+        "from argilla.metrics.text_classification import f1\n",
+        "\n",
+        "f1(name=\"banking77-topics-setfit\").visualize()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "oeV9Kig-tlrr"
+      },
+      "source": [
+        "## 🦾 Train a few-shot SetFit model\n",
+        "\n",
+        "Even if the zero-shot approach gives decent results (~0.86 F1), we can use the labelled dataset to train a few-shot model to obtain ~0.95 accuracy.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "BXcEBrtlzx_3"
+      },
+      "outputs": [],
+      "source": [
+        "# Load the handlabelled dataset from Argilla\n",
+        "ds = rg.load(\"banking77-topics-setfit\").prepare_for_training(train_size=0.8)\n",
+        "\n",
+        "# Load SetFit model from Hub\n",
```

Comment:
```suggestion
        "# Load SetFit model using a Sentence Transformer from the Hub\n",
        "# Feel free to experiment with other models, e.g. \"sentence-transformers/paraphrase-mpnet-base-v2\"\n",
```
The second comment is optional. The all-MiniLM model is ~90MB while the paraphrase is ~410MB. On some datasets, the paraphrase one outperforms the other by like 15% accuracy. I tend to use the paraphrase model for my personal tests.